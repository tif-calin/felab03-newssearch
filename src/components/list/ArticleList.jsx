import React from 'react';
import PropTypes from 'prop-types';

const ArticleList = props => {
  return <div className="ArticleList">

  </div>;
};

export default ArticleList;
