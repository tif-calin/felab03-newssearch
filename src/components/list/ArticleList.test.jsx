import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ArticleList from './ArticleList';

describe('ArticleList component', () => {
  afterEach(() => cleanup());

  it('renders ArticleList', () => {
    const { asFragment } = render(<ArticleList articles={[]}/>);
    expect(asFragment()).toMatchSnapshot();
  });
  
});
