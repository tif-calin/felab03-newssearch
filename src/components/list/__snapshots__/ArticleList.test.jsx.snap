// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList component renders ArticleList 1`] = `
<DocumentFragment>
  <ul
    class="ArticleList"
    role="list"
  />
</DocumentFragment>
`;
